generator/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
```

Comment:
we shouldn't use two spaces for indentation, as go uses tabs - I think I had `gofmt` in the script to make sure the code is correctly formatted before we commit it

Reply:
That is specifically for THIS json file... for some dumb reason.
https://github.com/OpenAPITools/openapi-generator-cli#configuration

Reply:
WTAF 🤦